wazo_dird/plugins/office365_backend/plugin.py

```diff
@@ -77,8 +77,11 @@ def search(self, term, args=None):
             microsoft_token = self._get_microsoft_token(**args)
         except MicrosoftTokenNotFoundException:
             return []
+        try:
+            contacts = self.office365.get_contacts(microsoft_token, self.endpoint)
+        except UnexpectedEndpointException:
```

Comment:
I think something should be logged here to help debugging when there's no result

Reply:
There is already a logger.error in the get_contacts method before the exception raising. Maybe we should only include more details? All it says is `An error occured while fetching information from microsoft endpoint`

Reply:
You're right I missed that. We could add the status code to the current log line and maybe log the response body in a logger.debug?

Reply:
I'll merge anyway since that's not something that changed from your PR.